src/pages/components/images.mdx

```diff
@@ -56,69 +52,88 @@ IBM Web Standards for all type, format, size, and other requirements.
+![Example of an illustration used in the image component with a matching background to the page. A light gray border separates the image background from the page background](../../images/component/image/image-modifier-border.png)
 
 </Column>
 </Row>
-<Caption>The border helps to separate the image from a background with the same color.</Caption>
+<Caption>The border helps to separate the illustration from a background with the same color.</Caption>
 
 <ResourceLinks name="Image" type="ui" multiComponent />
 
-### Content guidance for Image
+### Caption
+
+If an image would benefit from additional context, a caption can be added immediately beneath the media. Captions paired with imagery are useful in editorial experiences where the image cannot appear on its own or needs additional information.
+
+<Row>
+<Column colMd={8} colLg={8}>
+
+![Example of an image with a caption underneath. The image features a computer generated render of a quantum computer inside a laboratory. The caption underneath reads "A rendering of the first IBM Quantum System One on a university campus being installed at RPI."](../../images/component/image/image-modifier-caption.png)
+
+</Column>
+</Row>
+<Caption>Example of an image with an added caption underneath.</Caption>
 
-| Element        | Content type | Required | Instances | Character limit <br/>(English / translated) | Notes                                |
-| -------------- | ------------ | -------- | --------- | ------------------------------------------- | ------------------------------------ |
-| Image alt text | Text         | No       | 1         | 75/100                                      | Image description for accessibility. |
+### Lightbox
 
-## Image with caption
+The Lightbox property of an image enables users to click or tap on the image to display a larger version of the image inside a [Lightbox media viewer](https://www.ibm.com/standards/carbon/components/lightbox-media-viewer) overlay. If this option is enabled, a button will appear at the bottom corner of the image and the entire media will be clickable.
 
-With the Image with caption component, an image can include a caption beneath to provide more context. In addition, the
-image can be clicked to open a larger version of the image, using the
-[Lightbox media viewer](https://www.ibm.com/standards/carbon/components/lightbox-media-viewer) overlay.
+Inside the overlay, the caption becomes the heading and there is an opportunity for additional body copy. Lightbox is useful if the user would benefit from exploring more about the image itself, including the additional copy.
 
-Inside the overlay, the caption becomes the heading and there is an opportunity for additional body text.
+If both a border and Lightbox are enabled on a particular image, the Lightbox border will be displayed instead of the lighter border property. If Lightbox is enabled, the cursor will change to `zoom-in` mode while hovering to reinforce the click interaction.
```

Comment:
```suggestion
If both a border and lightbox are enabled on a particular image, the lightbox border will be displayed instead of the lighter border property. If lightbox is enabled, the cursor will change to `zoom-in` mode while hovering to reinforce the click interaction.
```